tests/foreman/api/test_errata.py

```diff
@@ -1430,3 +1468,253 @@ def rh_repo_module_manifest(module_sca_manifest_org, module_target_sat):
+            )['content-view']
+            cv_list.append(new_cv)
+
+    # total amount of CVs created matches expected and search results
+    assert len(cv_list) == 1 + (number_of_cvs * (number_of_lces + 1))
+    assert prior_cv_count + len(cv_list) == len(
+        target_sat.api.ContentView(organization=module_sca_manifest_org).search()
+    )
+    # one ak with newest CV and lce
+    host_lce = lce_list[-1].read()
+    host_cv = cv_list[-1].read()
+    ak = target_sat.api.ActivationKey(
+        organization=module_sca_manifest_org,
+        environment=host_lce,
+        content_view=host_cv,
+    ).create()
+    # content host, global registration
+    result = rhel8_contenthost.register(
+        org=module_sca_manifest_org,
+        activation_keys=ak.name,
+        target=target_sat,
+        loc=None,
+    )
+    assert result.status == 0, f'Failed to register the host: {rhel8_contenthost.hostname}'
+    assert rhel8_contenthost.subscribed
+    rhel8_contenthost.execute(r'subscription-manager repos --enable \*')
+    # Installing all outdated packages
+    pkgs = ' '.join(FAKE_9_YUM_OUTDATED_PACKAGES)
+    assert rhel8_contenthost.execute(f'yum install -y {pkgs}').status == 0
+    rhel8_contenthost.execute('subscription-manager repos')
+    # After installing packages, check available incremental updates
+    host = rhel8_contenthost.nailgun_host.read()
+    response = target_sat.api.Host().bulk_available_incremental_updates(
+        data={
+            'organization_id': module_sca_manifest_org.id,
+            'included': {'ids': [host.id]},
+            'errata_ids': FAKE_9_YUM_SECURITY_ERRATUM,
+        },
+    )
+    # expecting no available updates before CV change
+    assert (
+        response == []
+    ), f'No incremental updates should currently be available to host: {rhel8_contenthost.hostname}.'
+
+    # New Erratum CV filter created for host view
+    target_sat.api.ErratumContentViewFilter(content_view=host_cv, inclusion=True).create()
+    host_cv = target_sat.api.ContentView(id=host_cv.id).read()
+    lce_ids = sorted([lce.id for lce in lce_list])
+    # publish version with filter and promote
+    host_cvv = cv_publish_promote(
+        target_sat,
+        module_sca_manifest_org,
+        host_cv,
+        lce_ids,
+    )['content-view-version']
+
+    # cv is not updated to host yet, applicable errata should be zero
+    rhel8_contenthost.execute('subscription-manager repos')
+    host_app_errata = rhel8_contenthost.applicable_errata_count
+    assert host_app_errata == 0
+    # After adding filter to cv, check available incremental updates
+    host_app_packages = rhel8_contenthost.applicable_package_count
+    response = target_sat.api.Host().bulk_available_incremental_updates(
+        data={
+            'organization_id': module_sca_manifest_org.id,
+            'included': {'ids': [host.id]},
+            'errata_ids': FAKE_9_YUM_SECURITY_ERRATUM,
+        },
+    )
+    assert (
+        response
```

Comment:
Can't we check empty list here?
```suggestion
        response != []
```

Reply:
In prior reviews from Jake I've been told that it's good practice to just `assert response/result`, instead of asserting != specific empty value. @JacobCallahan any thoughts here?

Reply:
I'm thinking of the possibility of the response being `None`, if there were some unexpected issue, 
then `assert None != []` would pass, and some ambiguous value error would probably be had in an assertion below?

Reply:
correct, since we are expecting some kind of populated list, then a simple boolean assertion on the list itself should be fine.

Reply:
I will leave it upto you how to aseert, it is just an api call (`bulk_available_incremental_updates`) and output/response can assert with any recieved values inside dict.